### PR TITLE
Mobile push notifications

### PR DESCRIPTION
**Pull request by cjsauer**
Trello: https://trello.com/c/Vn56Ry3Q/2160-mobile-push-notifications

Parent PR: https://github.com/open-company/open-company-web/pull/979

### CODE REVIEW

**Comment by cjsauer** [comment]
[Mobile push notifications](https://trello.com/c/Vn56Ry3Q/2160-mobile-push-notifications)

**Comment by belucid** [comment]
@cjsauer CI failure here.

**Comment by cjsauer** [comment]
@belucid fixed

File: src/oc/notify/async/user.clj [hunk 1]
```diff
@@ -4,13 +4,14 @@
 
   Use of this is through core/async. A message is sent to the `user-chan`.
   "
-  (:require [clojure.core.async :as async :refer (>!! <!)]
-            [defun.core :refer (defun-)]
-            [taoensso.timbre :as timbre]
-            [oc.lib.db.pool :as pool]
+  (:require [clojure.core.async :as async :refer [<! >!!]]
+            [defun.core :refer [defun-]]
```
Comment (bago2k4) [inline src/oc/notify/async/user.clj]: Not sure if there is a specific reason why you changed these but we usually prefer round brackets for readability.
No need to change, just for the next time.

Sean made me notice it and i agree it's better.

Comment (cjsauer) [reply src/oc/notify/async/user.clj]: @bago2k4 oh it may have been my editor that automatically switched them when cleaning the ns declaration.

**Review by bago2k4 — APPROVED** [review]

**Pull request merged**